build/common.sh

```diff
@@ -1540,7 +1536,10 @@ function kube::release::docker::release() {
     # Activate credentials for the k8s.production.user@gmail.com
     gcloud config set account k8s.production.user@gmail.com
   fi
-  for arch in "${archs[@]}"; do
+      
+  for platform in "${KUBE_SERVER_PLATFORMS[@]}"; do
```

Comment:
No need to create another variable here.
for arch in " ${KUBE_SERVER_PLATFORMS[@]##*/}"